template-files/ios/dependencies.json

```diff
@@ -19,29 +7,10 @@
     "name": "Crashlytics",
     "version": "~> 3.13.4"
   },
-  {
-    "name": "FBAudienceNetwork",
-    "version": "5.5.0",
-    "otherPodfileFlags": ", :project_name => \"Facebook\""
-  },
-  {
-    "name": "FBSDKCoreKit",
-    "version": "~> 5.6.0",
-    "otherPodfileFlags": ", :project_name => \"Facebook\""
-  },
-  {
-    "name": "FBSDKLoginKit",
-    "version": "~> 5.6.0",
-    "otherPodfileFlags": ", :project_name => \"Facebook\""
```

Comment:
Is it safe to delete? Do you know why the project started to compile without that flag?

Reply:
I think it is safe. Isn't it just grouping subprojects together? I think it is a nice feature, but we would have to keep it in the Podfile to use it, which is no longer necessary imho.

Reply:
It has been introduced because otherwise `FBSDKLoginKit` wouldn't be able to private `FBSDKCoreKit` headers. That's why I commented — to ensure that it has been double checked this isn't the case